### PR TITLE
remove duplicate author

### PR DESCRIPTION
**Pull request by toh995**
### Description
Remove a duplicate author in the `AUTHORS` file. The author `Mike Bayer` was listed twice.

### Checklist
This pull request is:

- [x] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [ ] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

**Review by CaselIT — APPROVED** [review]

**Comment by CaselIT** [comment]
thanks!

File: AUTHORS [hunk 1]
```diff
@@ -20,7 +20,6 @@ Major contributing authors include:
 - Tony Locke
 - Hajime Nakagami
 - Vraj Mohan
-- Mike Bayer
 - Robert Leftwich
 - Taavi Burns
 - jonathan vanasco
```
Comment (toh995) [inline AUTHORS:25]: Also, this is random, but I'm not sure if the first and last name should be capitalized here?

**Comment by CaselIT** [comment]
I think that the names came the email of the commit authors, ordered by most commit.

I think that mike's name was duplicated because it had two different emails in the commit logs.

Personally I don't have any particular preference for the style here.

File: AUTHORS (hunk 1 above)
Comment (zzzeek) [reply AUTHORS:25]: sure that looks wrong, he uses capitals

Comment (zzzeek) [reply AUTHORS:25]: ill put this one in for now thanks!

**Pull request merged**